src/main/java/net/minecraftforge/event/world/jigsaw/ForgePillageOutpostPools.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Minecraft Forge
+ * Copyright (c) 2016-2019.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation version 2.1
+ * of the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package net.minecraftforge.event.world.jigsaw;
+
+import net.minecraftforge.event.ForgeEventFactory;
+
+public class ForgePillageOutpostPools {
+
+    public static void init() {
+    }
```

Comment:
You even call init at the apropiate time, so why does this class have a static initialiser?!?

Reply:
^^to fire the event only on time, because vanilla does the same. But I found a solution

Reply:
Your comment makes no sense... Though you did what we wanted (just without this init Method at all)